content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -225,3 +225,77 @@ const user = await prisma.user.findUnique({
   },
 })
 ```
+
+#### Filter on composite types
+
+<Admonition type="warning">
+
+Composite types are only available in MongoDB.
+
+</Admonition>
+
+In Prisma versions 4.3.0 and later, you can search on a multi-field constraint that contains composite types. For example, the following schema has a `Product` model, an embedded `Photo` composite type, and a multi-field unique constraint on the product `name` and photo `url`:
+
+```prisma file=schema.prisma
+model Product {
+  id          String  @id @default(auto()) @map("_id") @db.ObjectId
+  name        String  @unique
+  description String
+  photos      Photo[]
+
+  @@unique([name, photos.url]) // multi-field unique constraint
```

Comment:
Wondering if it's a good idea to also show a really simple constraint without using the multi-field. Users may think they can only use multi-field when they could also just do: 

```
@@unique([photos.url])
```

Reply:
Thanks. I've flagged this up in the text in two places, so I think it's clear now.